test/compliance.spec.ts

```diff
@@ -0,0 +1,56 @@
+import 'mocha';
+
+import * as _ from "lodash";
+import BN from "bn.js";
+import {Driver, DEPLOYMENT_SUBSET_MAIN} from "./driver";
+import chai from "chai";
+import {bn, evmIncreaseTime} from "./helpers";
+import {TransactionReceipt} from "web3-core";
+import {Web3Driver} from "../eth";
+
+chai.use(require('chai-bn')(BN));
+chai.use(require('./matchers'));
+
+const expect = chai.expect;
+
+describe('compliance-contract', async () => {
+
+    it('sets, gets, and updates validator compliance type', async () => {
+        // TODO see that committees are updates as a result of changing compliance
+
+        const d = await Driver.new();
+
+        const General = "General";
+        const Compliance = "Compliance";
```

Comment:
These constants can be put in the driver. they will be needed by other tests in the Elections contract

Reply:
RIght